Code/autopkglib/URLGetter.py

```diff
@@ -78,10 +78,11 @@ def produce_etag_headers(self, filename):
             self.existing_file_size = os.path.getsize(filename)
             etag = self.getxattr(self.xattr_etag)
             last_modified = self.getxattr(self.xattr_last_modified)
-            if etag:
-                headers["If-None-Match"] = etag
-            if last_modified:
-                headers["If-Modified-Since"] = last_modified
+            if not self.env["CHECK_FILESIZE_ONLY"]:
```

Comment:
We should probably use `if not self.env.get("CHECK_FILESIZE_ONLY"):` here in case it's undefined.

Reply:
That's very true.  I'll make that change and push it.

Reply:
Done in fbfc104.